zebra-chain/src/block/error.rs

```diff
@@ -0,0 +1,10 @@
+//! Errors that can occur when checking Block consensus rules.
+
+use thiserror::Error;
+
+#[allow(dead_code, missing_docs)]
+#[derive(Error, Debug, PartialEq)]
+pub enum BlockError {
+    #[error("invalid network upgrade")]
+    InvalidNetworkUpgrade,
+}
```

Comment:
Please name this error the same as the `zebra-consensus` error, and use the same description.

Reply:
https://github.com/ZcashFoundation/zebra/pull/2354/commits/5520611d46bea6cfaca662b4528ca0ddba64c9e1